config/requirements.txt

```
@@ -1,5 +1,6 @@
 check-python-versions==0.21.2
 Jinja2==3.1.2
 pyupgrade==3.6.0
+toml==0.10.2
```

Might it be that you were using Python 3.10 or earlier? On Python 3.11 it already has support for it.

We can do something like:

```suggestion
toml==0.10.2; python_version<"3.11"
```

---

Scratch that, indeed is a missing requirement 👍🏾